benchmarks/timeseries/benchmark_ringbuffer.py

```diff
@@ -30,12 +29,14 @@ def fill_buffer(
         # Push in random order
         for i in random.sample(range(MINUTES_IN_A_DAY), MINUTES_IN_A_DAY):
             buffer.update(
-                basetime + timedelta(days=day, minutes=i, seconds=i % 3),
-                element_type(i),
+                Sample(
+                    basetime + timedelta(days=day, minutes=i, seconds=i % 3),
+                    element_type(i),
```

Comment:
I guess only float will be supported from now on,  so we don't need the element_type.